pyzabbix/sender.py

```diff
@@ -385,3 +449,57 @@ def send(self, metrics):
         for m in range(0, len(metrics), self.chunk_size):
             result.parse(self._chunk_send(metrics[m:m + self.chunk_size]))
         return result
+
+    def _create_active_checks_request(self, host):
+        """Create a formatted request to zabbix from a host name.
+
+        :type host: str
+        :param host: The host to ask about
+
+        :rtype: str
+        :return: Formatted zabbix request
+        """
+        request = '{"request":"active checks","host":"%s"}' % host
```

Comment:
It's better use `format` method here.
But actually, I don't think we need this as separate function, because it's just 2-3 lines of code and it used just in one place. I think it should go to `send_active_checks`